app.rb

```diff
@@ -0,0 +1,140 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
```

Comment:
- [ ] I think it would be better if you fix the typo error you have when you are importing the `rentals` file. Right now, your application is crushing because of that. You should update the line 5 which is `require_relative 'rental'` to this `require_relative 'rentals'`